### DISCUSSION TITLE
Changes in a Contract Modification

### DISCUSSION THREAD

**Discussion by sorliden** [OP-TED/eForms-SDK#451] [Q&A] [answered]
- What types of changes (ChangedSectionIdentifier: CON-xxxx, LOT-xxxx, PROCEDURE, ...) are allowed in a Contract Modification?
- When should more than one ChangedSectionIdentifier be used (see example can-modif_24_duration.xml)?

**Comment by eduardsmirnov** [comment]
Hello publications office team, could you answer this question? It is a kind of show-stopper.

And additionally - why it is not allowed to have more than one <Change> element for a single contract inside one <ContractModification>?
What if we want to advertise two changes for the same contract? Is it expected to have a separate <ContractModification> element for every change we want to advert about a contract

**Reply by YvesJo** [reply]
There is no need to have as many elements as the number of modified fields. Section identifiers are "high level" ways to indicate the modified sections of the published notice. 
Text description is available to provide further information about the modifications without requiring the identification of every single modified field, just it should be clear enough to help the reader detect what has been modified.

**Comment by YvesJo** [comment, marked as answer]
I guess when speaking about "Change" you actually mean "Modification".
The section identifiers for a Modification actually refer to the sections of the published notice and you may expect the same identifiers as the ones for a Change (cf. https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html#_referring_to_sections_of_a_notice) with additionally the reference to the contract (CON-xxxx).
Multiple sections identifiers are to be used when, for a given contract, multiple sections of the published notice are modified.
